app/Notifications/NewCommentTag.php

```diff
@@ -48,6 +49,55 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     */
+    public function shouldSend(User $notifiable): bool
+    {
```

Comment:
Should have an is_modo check check to force send (if comment is not anon).

Reply:
Added.